Bottom: 3a455eb2744393fea1d7befeca0256d80d9f73ab
Top:    6ee88c4f0f59814eb1dadf471081d1c5b98814aa
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-16 23:59:18 +0000

hid/batch: Add missing #include "hid_draw.h"

I should have built-tested all HIDs when I refectored this.. oops.


---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index c464809..e62e313 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -13,6 +13,7 @@
 #include "data.h"
 #include "misc.h"
 #include "hid.h"
+#include "hid_draw.h"
 #include "../hidint.h"
 #include "pcb-printf.h"
